### ISSUE TITLE
Quantity support for iOS

### ISSUE THREAD

**Issue by Mwni** [daniele-pecora/nativescript-in-app-purchase#5]
Hi,

how hard would it be to implement support for specifying the quantity of a product when calling `order()`?
The quantity attribute has to be set on the SKPayment object [as described](https://developer.apple.com/documentation/storekit/skpayment). Inside this add on, the native payment object appears to held in the `nativeValue` field of the product `InAppProduct` class, but it is unclear where this native object exactly came from and how it is constructed.

**Comment by Mwni** [comment]
Okay so the `nativeValue` is set via https://github.com/daniele-pecora/nativescript-in-app-purchase/blob/cc9a0ceccd964727648fa3019bf87064547907de/src/in-app-purchase.ios.ts#L199

Let me try to modify the quantity value...

**Comment by Mwni** [comment]
Okay so that didn't work... I suppose it needs some special bindings to be able to set the native value via Javascript...

**Comment by Mwni** [comment]
Right, the quantity has to be set on `SKPayment`, I have mistakingly set it on the `SKProduct`. Setting it on `SKMutablePayment` appears to work, but does not reflect on the price shown in the iOS pay dialog. Not sure if that's by design.

**Comment by daniele-pecora** [comment]
@Mwni Feel free to send a PR.

>Not sure if that's by design.

As far as I know it will only work for consumable products.
And is also readonly:
https://developer.apple.com/documentation/storekit/skpayment/1506077-quantity

**Comment by Mwni** [comment]
I will send a PR once I can confirm setting this value actually affects the resulting purchase. At least in the testing environment, it appears that it does not. And then there's also the concern about making this cross compatible with android.
